chainercv/evaluations/eval_instance_segmentation_coco.py

```diff
@@ -0,0 +1,299 @@
+import itertools
+import numpy as np
+import os
+import six
+
+from chainercv.evaluations.eval_detection_coco import _redirect_stdout
+from chainercv.evaluations.eval_detection_coco import _summarize
+
+try:
+    import pycocotools.coco
+    import pycocotools.cocoeval
+    import pycocotools.mask as mask_tools
+    _available = True
+except ImportError:
+    _available = False
+
+
+def eval_instance_segmentation_coco(
+        pred_masks, pred_labels, pred_scores,
+        gt_masks, gt_labels, gt_areas=None, gt_crowdeds=None):
+    """Evaluate detections based on evaluation code of MS COCO.
+
+    This function evaluates predicted bounding boxes obtained from a dataset
+    by using average precision for each class.
+    The code is based on the evaluation code used in MS COCO.
+
+    Args:
+        pred_masks (iterable of numpy.ndarray): See the table below.
+        pred_labels (iterable of numpy.ndarray): See the table below.
+        pred_scores (iterable of numpy.ndarray): See the table below.
+        gt_masks (iterable of numpy.ndarray): See the table below.
+        gt_labels (iterable of numpy.ndarray): See the table below.
+        gt_areas (iterable of numpy.ndarray): See the table below. If
+            :obj:`None`, some scores are not returned.
+        gt_crowdeds (iterable of numpy.ndarray): See the table below.
+
+    .. csv-table::
+        :header: name, shape, dtype, format
+
+        :obj:`pred_masks`, ":math:`[(R, H, W)]`", :obj:`float32`, \
+        ":math:`(y_{min}, x_{min}, y_{max}, x_{max})`"
+        :obj:`pred_labels`, ":math:`[(R,)]`", :obj:`int32`, \
+        ":math:`[0, \#fg\_class - 1]`"
+        :obj:`pred_scores`, ":math:`[(R,)]`", :obj:`float32`, \
+        --
+        :obj:`gt_masks`, ":math:`[(R, H, W)]`", :obj:`float32`, \
+        ":math:`(y_{min}, x_{min}, y_{max}, x_{max})`"
+        :obj:`gt_labels`, ":math:`[(R,)]`", :obj:`int32`, \
+        ":math:`[0, \#fg\_class - 1]`"
+        :obj:`gt_areas`, ":math:`[(R,)]`", \
+        :obj:`float32`, --
+        :obj:`gt_crowdeds`, ":math:`[(R,)]`", :obj:`bool`, --
+
+    All inputs should have the same length. For more detailed explanation
+    of the inputs, please refer to
+    :class:`chainercv.datasets.COCOInstanceSegmentationDataset`.
+
+    .. seealso::
+        :class:`chainercv.datasets.COCOInstanceSegmentationDataset`.
+
+    Returns:
+        dict:
+
+        The keys, value-types and the description of the values are listed
+        below. The APs and ARs calculated with different iou
+        thresholds, sizes of objects, and numbers of detections
+        per image. For more details on the 12 patterns of evaluation metrics,
+        please refer to COCO's official `evaluation page`_.
+
+        .. csv-table::
+            :header: key, type, description
+
+            ap/iou=0.50:0.95/area=all/max_dets=100, *numpy.ndarray*, \
+                [#coco_det_eval_1]_
```

Comment:
change `coco_det_eval_1` to `coco_inst_segm_eval_1`.
The link conflicts with `eval_detection_coco`.
You will notice the problem after compiling.